lib/db.c

```diff
@@ -8133,7 +8133,7 @@ del_hook(grn_ctx *ctx, grn_obj *obj, grn_hook_entry entry, grn_id target_id)
     if (hook->proc) {
       continue;
     }
-    if (hook->hld_size != sizeof(grn_obj_default_set_value_hook_data)) {
+    if (hook->data_holder_size != sizeof(grn_obj_default_set_value_hook_data)) {
```

Comment:
How about simplifying `data` to something like `holder_size` or replacing it with `metadata` because I'm thinking `data` is a rather generic name here?